### DISCUSSION TITLE
Content for DM Section - The Role of a Delivery Manager

### DISCUSSION THREAD

**Discussion by torihutchinson** [nhsbsa/nhsbsa-digital-playbook#258] [Delivery]
### The Role of a Delivery Manager 

Emily Webber, the former Head of Agile Delivery for the Government Digital Service (GDS), offered a valuable starting point in defining the Delivery Manager role. She defined the role as:

>  ‘The servant leader, who keeps pace with the introduction of relevant agile and lean tools and techniques and removes obstacles and blockers that might get in the way of delivery.’ The definition breaks down the role into three distinct areas and establishes what the role is accountable for: 

**Agile and Lean Practices** 

You are responsible for ensuring that the team are applying agile and lean practices. 

**Team Health and Happiness** 

You are responsible for making sure the team has everything they need to be productive, that you have the right make-up of capabilities to deliver what is needed and implementing the conditions to create the right environment for effective delivery.  

**Delivery Support** 

You are responsible for ensuring the team have everything they need to do their job.  

 

**Delivery Managers are ultimately accountable for the performance of the team and the resultant delivery of successful outcomes for a service or product.** 

 
### Delivery Management at the NHSBSA

We have multiple roles in our Delivery Management job family, which are accountable for different levels of the DDaT delivery portfolio.  

- **Heads of Delivery and Consultancy** – accountable for the end-to-end delivery, and management of, an entire Directorate portfolio of DDaT services and products. For example, Citizen Services, Enterprise and DDaT Services, Primary Care Services, Workforce.  
- Partnering at a Director and Head of Service level to shape vision and strategic objectives for area of accountability.  

- **Lead Delivery Managers** - accountable for the end-to-end delivery, and management of, an sub-portfolio of DDaT services and products or a particularly large and/or complex service. For example, NHS Jobs, all services under Help with Health Costs, Microsoft products.  
- Partnering at a Head of Service level to shape roadmap for area of accountability.  

- **Delivery Managers** - lead multidisciplinary Delivery Teams a to deliver products and services which meet the needs of our users and contribute to achievement of strategic objectives for that area.  
- Partnering with Product Owners to create delivery plan for area of accountability.  

- **Junior Delivery Managers** - the role of a Junior is aligned with the role of Delivery Manager (DM), with the ability to perform all responsibilities within a supported learning environment and mentorship from their DM and Lead DM. 

Please see below for more information on each role. 

**Delivery Manager Accountability** 

Delivery Managers are accountable for the performance of their team and the resultant delivery of successful outcomes for a service or product. 

**Team, Collaboration & Communication** 

- Creation and maintenance of an efficient, self-organising team  
- Delivery team member onboarding, inductions, knowledge transfer and kick-off  
- Ensures teams plan appropriately for their own capacity, helping to dynamically organise delivery activities at a detailed level 
- Facilitating the right ‘mindset’ of the team, encouraging full participation and commitment from all team members  
- Management of Delivery Team worries and/or conflict, and escalation, if appropriate  
- Protect the team and make sure the team collaborates, communicates, and focuses on what is most important, keeping the team focused on priorities as agreed by the Service Manager or Product Owner  

**Delivery, Planning & Strategy** 

- Understands the environment and can prioritise the most important or highest value tasks 
- Alignment of sprint goals to overall project, programme, and strategic objectives  
- Feed info into business planning, business case review, quarterly delivery plans, service digital KPI reporting and benefits realisation tracking 
- Responsible for delivering milestones on the product roadmap 
- Creating a productive environment, setting a good, sustainable delivery pace 
- Proactively manage internal and external complex dependencies, overcome obstacles/blockers, and get the best value against constraints 

**Reporting, Governance & Compliance** 

- Tracking and reporting of progress 
- Building effective working relationships across customers and stakeholders  
- Ensuring budget is in place and on track  
- Be accountable for governance within the team, working alongside Project Management if applicable 
- Risk and issue management 
- Identify and secure the right resources, aligned with delivery requirements 
- Create and submit resource SOW’s, working with Commercial 
- Approve Technical SOWs from a cost perspective 
- Submitting and management of change requests to enable deployment releases 
- Making sure accessibility is factored into each feature or activity the team’s working on 
- Agile Methodology, Communities of Practice & Coaching 
- Adhere to and promote agile principles, setting up the agile environment your team needs to build and iterate a user-centred service 
- Coach team members and others, facilitate continuous improvement and apply the most appropriate agile and lean tools and techniques for their environment 
- Facilitation of ceremonies – daily stand-up, retrospective, sprint review, sprint planning and backlog refinement   
- Integrate efficiently with other (agile or non-agile) delivery teams 
- Communicating between the technical and non-technical, able to listen to the needs of technical and business stakeholders and interpret them 
- Engagement with Professional Leads 
- Participation in Delivery Manager Community and sharing of learning  
- Coaching and mentoring of a Junior DM where applicable 
- Deputise for LDDM  
- On-going learning, personal and professional development, and the development of the Portfolio function as a whole 

 
**Role of a Junior Delivery Manager** 

The role of a Junior is aligned with the role of Delivery Manager (DM), with the ability to perform all responsibilities within a supported learning environment and mentorship from their DM and Lead DM.  

**Team, Collaboration & Communication** 

- Support and maintenance of an established efficient, self-organising team  
- Delivery team member onboarding, inductions, knowledge transfer and kick-off  
- Ensures teams plan appropriately for their own capacity, helping to dynamically organise delivery activities at a detailed level 
- Facilitating the right ‘mindset’ of the team, encouraging full participation and commitment from all team members  
- Management of Delivery Team worries and/or conflict, and escalation, if appropriate  
- Protect the team and make sure the team collaborates, communicates, and focuses on what is most important, keeping the team focused on priorities as agreed by the Service Manager or Product Owner  

**Delivery, Planning & Strategy** 

- Understands the environment and can prioritise the most important or highest value tasks 
- Alignment of sprint goals to overall project, programme, and strategic objectives  
- Feed info into business planning, business case review, quarterly delivery plans, service digital KPI reporting and benefits realisation tracking 
- Responsible for delivering milestones on the product roadmap 
- Creating a productive environment, setting a good, sustainable delivery pace 
- Overcome obstacles/blockers and get the best value against constraints 

**Reporting, Governance & Compliance** 

- Tracking and reporting of progress 
- Building effective working relationships across customers and stakeholders  
- Forecast and track delivery costs and benefits  
- Be accountable for governance within the team, working alongside Project Management if applicable 
- Risk and issue management 
- Identify and feedback on resource requirements  
- Submitting and management of change requests to enable deployment releases 
- Making sure accessibility is factored into each feature or activity the team’s working on 

 **Agile Methodology, Communities of Practice & Coaching** 

- Adhere to and promote agile principles, setting up the agile environment your team needs to build and iterate a user-centred service 
- Coach team members and others, facilitate continuous improvement and apply the most appropriate agile and lean tools and techniques for their environment 
- Facilitation of ceremonies – daily stand-up, retrospective, sprint review, sprint planning and backlog refinement   
- Integrate efficiently with other (agile or non-agile) delivery teams 
- Communicating between the technical and non-technical, able to listen to the needs of technical and business stakeholders and interpret them 
- Engagement with Professional Leads (currently not applicable to Insight) 
- Participation in Delivery Manager Community and sharing of learning  
- Deputise for DM  
- On-going learning, personal and professional development, and the development of the Portfolio function as a whole

**Comment by Rathi96** [comment]
Looking to enhance your product delivery process? Check out this resource on managing spilled stories for better predictability here. It’s a game-changer! https://www.kaiburr.com/blog/how-to-improve-predictability-of-product-delivery-by-managing-spilled-stories/
